snapshot-labs/sx-monorepo
# feat: add new privacy type

### Summary
- Adds new option in space settings to select "any" privacy type
- Adds new option on proposal editor to enable/disable shielded voting 
- Adds new option on update proposal editor to enable/disable shielded voting

Towards: https://github.com/snapshot-labs/workflow/issues/277

### How to test

2. Go to your settings, under Privacy - you should see the new option "any"
<img width="445" alt="Untitled 4" src="https://github.com/user-attachments/assets/2390c35a-3225-4956-8e76-abf1cb24e3af">

3. Inside proposal editor, you should see new option
![image](https://github.com/user-attachments/assets/77038fb9-795e-4fdb-9d38-6f88ba6c415c)
<img width="292" alt="image" src="https://github.com/user-attachments/assets/26f73f70-bdcf-4376-a9a1-875e08fc758e">

4. Try create a new proposal with/without shutter
5. Try updating proposal with/without shutter
